src/backend/distributed/planner/multi_logical_planner.c

```diff
@@ -130,6 +130,8 @@ static MultiNode * ApplyCartesianProduct(MultiNode *leftNode, MultiNode *rightNo
  * Local functions forward declarations for subquery pushdown. Note that these
  * functions will be removed with upcoming subqery changes.
  */
+static Node * ReplaceExternParamsInOriginalQuery(Node *inputNode,
```

Comment:
I don't think it's a good idea to say "originalQuery" here.  This is a pretty general facility.  Maybe "ResolveExternalParams" or "InstantiateExternalParamsAsConsts"?